compiler/rustc_feature/src/unstable.rs

```diff
@@ -432,8 +432,8 @@ declare_features! (
     (unstable, deprecated_suggestion, "1.61.0", Some(94785)),
     /// Allows deref patterns.
     (incomplete, deref_patterns, "1.79.0", Some(87121)),
-    /// Allows deriving `SmartPointer` traits
-    (unstable, derive_smart_pointer, "1.79.0", Some(123430)),
+    /// Allows deriving `CoerceReferent` traits
```

Comment:
I believe you should register `derive_smart_pointer` as a removed feature.

Reply:
Applied. I moved it into the `removed` list.